example/index.html

```diff
@@ -11,6 +11,16 @@
     <script>
         document.documentElement.className = document.documentElement.className.replace("no-js", "js");
     </script>
+    <script>
+    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+    })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+
+    ga('create', 'UA-40614522-2', 'auto');
```

Comment:
This should use a dedicated property (under the springload.github.io account?). We don't want our debugging property IDs to be available publicly.